CHANGELOG.md

```diff
@@ -5,8 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
-  - nothing yet
+## Unreleased [3.5.2]
+### Changed
+- Removed memory leaks that were identified by the code sanitizer
```

Comment:
I think the updates might be better under fixes: "Fixed memory leaks identified by code sanitizer"

Reply:
Now updated in commit b404ae159df68506f4266529b0a6236ac416b1c9